src/com/palmergames/bukkit/towny/Towny.java

```diff
@@ -168,7 +170,7 @@ public void onEnable() {
 			addMetricsCharts();
 		} catch (TownyInitException tie) {
 			addError(tie.getError());
-			getLogger().log(Level.SEVERE, tie.getMessage(), tie.getStackTrace());
+			getLogger().log(Level.SEVERE, tie.getMessage(), tie);
```

Comment:
Do we not want the stacktrace for problem solving? What is the tie going to show the log?

Reply:
Using getStackTrace() sent it to a different method which thought these were log parameters, basically ignoring it.